### DISCUSSION TITLE
Is it possible to hide the documentation endpoint completely?

### DISCUSSION THREAD

**Discussion by enilsen16** [fastapi/fastapi#7526] [Questions] [answered]
I couldn't seem to find any info on how to hide the interactive docs in a production environment.

For example, We use the documentation locally to test and for other devs to refer to what's expected but this isn't something we want to expose in production. Is there a global way to hide documentation?

**Comment by phy25** [comment, marked as answer]
Have you read https://fastapi.tiangolo.com/tutorial/metadata/#docs-urls?

**Comment by enilsen16** [comment]
Awww yes, thank you!

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

Thanks for reporting back and closing the issue @enilsen16  :+1:
